shell/pages/c/_cluster/settings/performance.vue

```diff
@@ -23,6 +23,7 @@ export default {
   async fetch() {
     try {
       this.uiPerfSetting = await this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.UI_PERFORMANCE });
+      this.authUserTTL = await this.$store.dispatch(`management/find`, { type: MANAGEMENT.SETTING, id: SETTING.AUTH_USER_SESSION_TTL_MINUTES });
```

Comment:
This try/catch is specific to the UI_PERFORMANCE setting request. If the request to fetch AUTH_USER_SESSION_TTL_MINUTES fails then the former gets reset to default. Can it be in it's own try/catch with a fall back value?

Reply:
Correct. Though thinking about it if we can't fetch a `authUserTTL` value we shouldn't set a default and then not validate for it.